site/components/form/Map.tsx

```diff
@@ -1,22 +1,62 @@
-import { CSSProperties, ReactElement } from "react";
-import MapBox, { ViewState } from "react-map-gl";
+import uniqueId from "lodash/uniqueId";
+import { CSSProperties, ReactElement, ReactNode } from "react";
+import MapBox, { Marker, ViewState } from "react-map-gl";
+import { Stop } from "../../schemas/consequence.schema";
 
 interface MapProps {
     initialViewState: Partial<ViewState>;
     style: CSSProperties;
     mapStyle: string;
+    selected?: Stop[];
+    searched?: Stop[];
+    inputId?: keyof Stop;
 }
 
-const Map = ({ initialViewState, style, mapStyle }: MapProps): ReactElement | null => {
+const Map = ({ initialViewState, style, mapStyle, selected, searched }: MapProps): ReactElement | null => {
     const mapboxAccessToken = process.env.MAP_BOX_ACCESS_TOKEN;
 
+    const getMarkers = (selected: Stop[], searched: Stop[]): ReactNode => {
+        const inTable =
+            selected && selected.length > 0
+                ? selected.map((s: Stop) => (
+                      <Marker
+                          key={uniqueId(s.atcoCode)}
+                          longitude={Number(s.longitude)}
```

Comment:
Rather than converting to numbers here, I'm thinking we could just use zod to coerce the value into a number at the point of parsing the stops, the schema for longitude and latitude can be updated to `z.coerce.number()`

Reply:
done